Automattic/jetpack
# Forms: Fix hash generation

## Proposed changes:
* Attempt to fix Forms hash generation causing an issue reported here: p1683922710395059-slack-C0299DMPG

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post a add a Form block
* Wrap some fields in the Group block and use the Sidebar to change the Group's block spacing
* Publish the page, go to the published page, fill out the form, and submit it
* You should the the after-submission form summary
